README.rst

```diff
@@ -28,133 +28,43 @@ Nengo: Large-scale brain modelling in Python
 Installation
 ============
 
-To install Nengo, use::
+To install Nengo::
 
   pip install nengo
 
 Nengo depends on `NumPy <http://www.numpy.org/>`_.
-If you have difficulty installing,
-try installing NumPy first (see below).
+If you have difficulty installing Nengo, try
+`installing NumPy <https://pythonhosted.org/nengo/getting_started.html#installation>`_ first.
```

Comment:
This link isn't Numpy-specific. Maybe put it further up (possibly in addition to here), or label it differently, since it has information that might be useful even if you don't have problems with Numpy.